### ISSUE TITLE
Windows: installer created via Gitian is not deterministic

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/omnicore#125]
Just to keep track of it.

I stumbled upon this:

https://github.com/bitcoin/bitcoin/commit/6083295f4666a9527ec3311f1acaea941bf7bb86

```
Update bitcoin icon to make nsis setup exe deterministic.

This adds a 32x32 16-bit icon to the bitcoin.ico file.
Though this realistically probably looks worse than the 32-bit
32x32 being displayed on a 16-bit monitor, it makes the nsis
setup exe deterministic in gitian output (go figure) which makes
the slight visual loss for users of very old monitors/computers
worth it.
```

**Comment by zathras-crypto** [comment]
Nice find, let me take another look at the icon file :)

**Comment by dexX7** [comment]
Resolved via #126.
